File: boj8772.py
'''
백준 8772번
Szyfr Cezara
BRONZ 2
싸이퍼 암호
'''
import sys
input = sys.stdin.readline

T = int(input())
while T:
    T -= 1
    z = input()
    string = input().rstrip()
    char = list(input().rstrip())
    # print(char)
    diff = (ord(char[0])-ord(string[0]) + 26) % 26
    for c in string[1:]:
        num = ord(c) + diff
        if num > 122:
            num %= 122
            num += 96
        char.append(chr(num))
    print(''.join(char))

File: boj8772.cpp
/*
백준 8772번
Szyfr Cezara
BRONZ 2
싸이퍼 암호
*/

#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;
int main() {
    ios::sync_with_stdio(0), cin.tie(0), cout.tie(0);
    int t;
    cin >> t;
    while (t--) {
      int a;
      string s;
      string c;
      cin >> a >> s >> c;
      int diff = (c[0] - s[0] + 26) % 26;
      for (int i = 1; i < s.length(); i++) {
          int num = s[i] + diff;
          if(num > 122){
              num %= 122;
              num += 96;
          }
          c.push_back(char(num));
      }
      cout << c << '\n';
    }
    return 0;
}